docs/tutorial/batch_predict/batch_predict.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>Is there a way to publish this, so we won't use development?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/2415/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<p>In progress</p>